### PR TITLE
Completed css-flexbox exercise. - Rosemary Liu

### PR DESCRIPTION
**Pull request by rosemary-liu**
image
image
image
image

### CODE REVIEW

File: css-flexbox/display-flex.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - display: flex</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+
+  </style>
```
Comment (TimelikeClosure) [inline css-flexbox/display-flex.html:13]: Please finish the `display: flex` part of the exercise.

File: css-flexbox/flex-wrap.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+
+  </style>
```
Comment (TimelikeClosure) [inline css-flexbox/flex-wrap.html:13]: Please finish the `flex: wrap` part of the exercise.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: css-flexbox/justify-content.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - justify-content</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="pokemon.css">
+  <style>
+    /* your styles here 👇 */
+
+    #card-deck {
+      justify-content: center;
```
Comment (Zhiwutian) [inline css-flexbox/justify-content.html:14]: justify content will not work without `display:flex`

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
quick update needed

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**